lib/cts/cases/create.rb

```diff
@@ -366,30 +372,32 @@ def get_journey_for_flagged_state(flag)
     end
 
     def responder
-      @responder ||= if !options.key?(:responder)
-                       if responding_team.responders.empty?
-                         raise "Responding team '#{responding_team.name}' has no responders."
-                       else
-                         responding_team.responders.first
-                       end
-                     else
-                       CTS::find_user(options[:responder])
-                     end
+      @responder ||= begin
+        if options.key?(:responder)
+          CTS::find_user(options[:responder])
+        else
+          responding_team.responders.first!
+        end
+      end
     end
 
     def responding_team
-      @responding_team ||= if !options.key?(:responding_team)
-                             if options.key?(:responder)
-                               responder.responding_teams.first
-                             else
-                               BusinessUnit.responding.sample
-                             end
-                           else
-                             CTS::find_team(options[:responding_team])
-                           end
+      @responding_team ||= begin
+        if !options.key?(:responding_team)
```

Comment:
while you're here, it would be good to flip the ! again. I'm amazed that rubocop doesn't detect this - !x feels like an anit-pattern in every language that I have ever used as it is so small and hard to spot

Reply:
Ok

Reply:
I took this comment to mean I have liberty to reformat the entire file. Sorry.